frinx-uniconfig/user-guide/operational-procedures/data-security-models/index.md

```diff
@@ -226,14 +227,73 @@ does not already provide encryption capability, then encrypted-paths module is u
 installation of device ('netconfReferenceModuleName' and matching of revisions are ignored).
 !!!
 
+### Change encryption status
+
+!!!
+For proper working of this RPC it is necessary to enable notifications with this parameter:
+
+```settings
+notifications.enabled=true
+```
+!!!
+
+There is one parameter which enable or disable encryption can be changed by 'change-encryption-status' 
```

Comment:
Encryption can be enabled or disabled using 'change-encryption-status' RPC request:

Reply:
done